addons/purchase/views/purchase_views.xml

```diff
@@ -769,7 +764,7 @@
                 action = records._send_reminder_mail(send_single=True)
         </field>
     </record>
-    
+
```

Comment:
:warning: There is some useless diff due to your linter. Idk if `purchase` is ok with such things. Personally I don't care :p